### ISSUE TITLE
[Feature request] Auto delete watched

### ISSUE THREAD

**Issue by Baardmans** [razzeee/script.trakt#460]
I'd like to be able to set automatic deletion from certain lists. For example when I've watched a movie that's on my Watchlist, remove it in order to keep the list clean / up to date.

**Comment by rectifyer** [comment]
Trakt already auto removes items from your watchlist after you watch them. It is a core Trakt feature.

**Comment by Baardmans** [comment]
@rectifyer, yeah, but this only works on the Watchlist though, I'd like this feature on other lists as well.

**Comment by razzeee** [comment]
Well that should be setup on the website site of things and not in the kodi addon, if you ask me.
